src/Core_Sideloader_Studio/UniversalAutoResolver/Core.UAR.Studio.cs

```diff
@@ -359,6 +369,9 @@ internal static void ResolveStudioFilter(ExtensibleSaveFormat.PluginData extData
             if (extData != null && extData.data.ContainsKey("filterInfoGUID"))
             {
                 string filterGUID = (string)extData.data["filterInfoGUID"];
+                string filterAuthor = (string)extData.data["filterInfoAuthor"];
+                string filterWebsite = (string)extData.data["filterInfoWebsite"];
```

Comment:
Oops, one of these days I'll remember dictionaries store nulls and therefore throw exceptions on not found instead of return null...but not this day...